lib/presentation/widget/login/login_widget.dart

```diff
@@ -114,45 +116,55 @@ class _LoginWidgetState extends State<LoginWidget> {
                         ),
                         Padding(
                           padding: EdgeInsets.symmetric(horizontal: 40),
-                          child: TextField(
-                              onChanged: (value) => viewModel.setUrlText(value),
-                              style: TextStyle(
-                                  color: AppColor.loginTextFieldTextColor),
-                              textInputAction: TextInputAction.next,
-                              decoration: textFieldUtil.loginInputDecoration(
-                                  AppLocalizations.of(context)
-                                      .stringOf('https'))),
+                          child: StoreConnector<AppState, dartz.Either<Failure, Success>>(
```

Comment:
Why Aliase ? There is an other `Either `?

Reply:
No, it seems to have the same `State` class

Reply:
Confirmed, there are 2 differents `State` class. 1 from `dartz`, 1 from `core widget` library.